vault/dendron.release.2021-11-02.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 4q0nOxVnDvAMD18zyNebk
+title: 'Release Notes (version 0.66)'
+desc: ''
+updated: 1635872354186
+created: 1635870362508
+---
+
+Dendron 0.66 has sprouted  🌱
+
+An early version of task notes is now available to test out! Take a look and feel free to provide feedback on [RFC 13 - Todo Notes in the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358).
```

Comment:
This release brings native support to managing tasks in Dendron. Task management is one of the most asked for features in Dendron. Please try it out and provide any feedback in [the Task Notes GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)

- NOTE: task notes are considered a [[seed feature|dendron://dendron.dendron-site/tags.stage.seed]] which means its still experimental and the interface is subject to change.

Reply:
we're working on slowly introducing the  [[Dendron Maturity Levels|dendron://dendron.dendron-site/dendron.ref.maturity]]  in existing features and when describing release features